### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by mrs1669**
> MemoryGar `gabe` CollectorSettings -> MemoryGarbageCollectorSettings

and fix typos in ReleaseToolong & scripts.

### CODE REVIEW

File: ReleaseTooling/Sources/PodspecsTester/InitializeSource.swift [hunk 1]
```diff
@@ -197,7 +197,7 @@ enum InitializeSpecTesting {
     return versionMatches[0][1]
   }
 
-  private static func createPodDirctory(specRepoPath: String, podName: String,
+  private static func createPodDirectory(specRepoPath: String, podName: String,
                                         version: String) -> URL {
```
Comment (paulb777) [inline ReleaseTooling/Sources/PodspecsTester/InitializeSource.swift:201]: Looks like the indentation needs to be adjusted here to fix clang-format failure.

**Review by paulb777 — COMMENTED** [review]
Thanks! Looks like one adjustment needed to fix CI.

File: ReleaseTooling/Sources/PodspecsTester/InitializeSource.swift (hunk 1 above)
Comment (mrs1669) [reply ReleaseTooling/Sources/PodspecsTester/InitializeSource.swift]: thanks! 
I've adjusted the indentation to resolve the clang-format error. Let me know if anything else needs to be changed.

https://github.com/firebase/firebase-ios-sdk/pull/12793/commits/c2e03950d94b8dd8b62220bba36d874a74b8d4d4

**Review by paulb777 — APPROVED** [review]

**Review by paulb777 — APPROVED** [review]
Thanks!

**Pull request merged**